state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -406,26 +402,21 @@ pub mod pallet {
 						});
 						// Handle swap completion logic.
 						match &swap.swap_type {
-							SwapType::Swap(destination_address) =>
-								if !egress_amount.is_zero() {
```

Comment:
I guess this is the reason we use strictly greater? Just so we can avoid explicitly checking for zero?

Reply:
Correct